static/app/views/performance/transactionSummary/transactionTags/tagValueTable.tsx

```diff
@@ -238,8 +238,11 @@ export class TagValueTable extends Component<Props, State> {
     }
 
     if (column.key === 'action') {
+      const searchConditions = tokenizeSearch(eventView.query);
+      const disabled = !!searchConditions.hasTag(dataRow.tags_key);
```

Comment:
@k-fish the return type of `hasTag` should be fixed in a follow up PR

Reply:
Nah I'll just fix it now 😎